### ISSUE TITLE
Phase 1/3 become slow 

### ISSUE THREAD

**Issue by jama9999** [madMAx43v3r/chia-plotter#828]
![Снимок экрана (56)](https://user-images.githubusercontent.com/77139949/125677201-cf879e9f-7e66-4e5a-981f-4e72651392ff.png)
![Снимок экрана (57)](https://user-images.githubusercontent.com/77139949/125678359-c8cf3d7f-41fb-4e21-88b5-25935ae4c6af.png)

First 2 weeks plotter was making a plot in 5k seconds and suddenly started to slow down and making a plot in 7.5/8k secs.
I have 2 similar setups and had the same problem in both. This problem occurred on the second computer a day later. 
Only 2nd phase as fast as it was.

My setups:
1st
CPU - Xeon 2690
M.2 NvMe - 1 tb 1800 mb/s write speed
RAM - 16 gb 1333mhz

2nd
CPU - Xeon 2689
M.2 NvMe - 512 gb 3200 mb/s write speed
RAM - 16 gb 1333mhz

Plotting time is the same in both setups!!

**Comment by endurance1968** [comment]
So there were no soft or hardware changes on your end comparing the times or did you replace the plotter version?
Did you check your SSDs for Activity times - maybe they are running out of max TB written?

**Comment by jama9999** [comment]
> So there were no soft or hardware changes on your end comparing the times or did you replace the plotter version?
> Did you check your SSDs for Activity times - maybe they are running out of max TB written?

Nothing was changed. I even replaced my ssd to the new one and faster one to solve the problem but it didnt work

**Comment by daveooo11** [comment]
Potentially related to #785. I have experienced the same exact proportional slowdown on the same phases, in windows as well. Tried in different versions of MM and it doesn't seem to fix it either. It was just roughly around the time the contract address enabled plotter came out this started happening.

**Comment by PoizenJam** [comment]
Experiencing the same issues myself.

I plot on windows 10 using 2 x 1TB NVME (as T1 and T2) drives with a 10700k. Used to achieve plot times of 48-50min. Over the last 24 hours my P1/3 times have progressively nearly doubled.

I cannot figure out why: I have restarted, I have trimmed both drives, I have tried 0.1.1 and 0.1.2a (running standalone or via PSChiaplotter), and I have checked that there is no wear-induced loss of speed (Crystaldisk info reports no speed decrease, health is ~%90). Also no thermal throttling reported on any devices.

When I observe the behaviour, it's pretty odd. CPU Just seems to keep spiking during P1/3 but doesn't stay pinned like it used to. Weirdly T1 usage also seems more intermittent, just like CPU. Table 1 writes quick but then performance drops off a cliff. P2/4 seem mostly unaffected by this degradation.
